packages/material-ui/src/ExpansionPanel/ExpansionPanel.js

```diff
@@ -181,6 +182,10 @@ ExpansionPanel.propTypes = {
    * @ignore
    */
   className: PropTypes.string,
+  /**
+   * The component used for the collapse effect.
+   */
+  CollapseComponent: PropTypes.elementType,
```

Comment:
What about just calling this `TransitionComponent`? 

Current implementation would allow for `Fade` or `Grow` as far as I can tell. Makes possible use cases more obvious.

Reply:
😱 yes!

Reply:
Done in https://github.com/mui-org/material-ui/pull/14617/commits/08428aaf635969a5fc5cf1cea1c28578b16ed1fd